File: Search/Linear/LinearSearch.py
def LinearSearch(arr,n,num):
    for i in range(0,n):
        if(arr[i]==num):
            print("Element Present at place ",i+1)
            return
    print("Element Not Found")
    return         

arr=[1,2,3,4,5,10]
num=10
size=len(arr)
LinearSearch(arr,size,num)

File: Search/Linear/LinearSearch.js
function LinearSearch(arr,n,x){
    for(let i=0;i<n;i++){
        if(arr[i]==x){
            console.log(`Element found at place ${i+1}`)
            return
        }
    }
    console.log("Element Not Found")
    return
}

let arr=[2,3,4,5,6,10]
let num=10
LinearSearch(arr,arr.length,num)

